build.sbt

```diff
@@ -206,7 +206,8 @@ lazy val core = (project in file("core"))
       "io.lemonlabs"     %% "scala-uri"       % "0.5.7",
       "net.virtual-void" %% "json-lenses"     % "0.6.2" excludeAll
         ExclusionRule(organization = "org.parboiled", name = "parboiled-scala_2.11"),
-      "org.typelevel" %% "cats-effect" % "1.4.0"
+      "org.typelevel" %% "cats-effect" % "1.4.0",
```

Comment:
1.4.0 is old, consider 2.1.4

Reply:
cats-effects 1.4.0 is the latest version on the 1.x line and the same version we are using in the rest of the internal components.
Updating to 2.x is probably a future task but it would need more investigation to see that everything that depends on jsonapi-scala still works. Leaving that for another PR.